CODE_OF_CONDUCT.md

```diff
@@ -18,47 +48,63 @@
-* No conduct intended to stalk, harass, or intimidate other users.
-* Do not continue to engage in conversation with a user that has specifically requested for said engagement to cease regardless of platform-specific privacy tools employed.
-* No inciting others to engage another user in continued interaction or discussion after a user has requested for said engagement with that user to cease.
-* No xenophobia or violent nationalism.
+There is also going to be behaviour that will be more strictly prohibited.
+
+### Discrimination
+
+* Do not participate in discrimination or comments promoting or reinforcing the existing systems of oppression of any groups or people based on gender, gender expression, race, ethnicity, nationality, sexuality, religion, disability, mental illness, neurodivergence, personal appearance, physical appearance, body size, age, or class.
 * Do not claim “reverse-isms,” e.g. “reverse racism”
-* No sexual or violent imagery without a content warning.
-* No sexualized depictions of minors, in any way. (This includes drawings and 3D renders).
-* No aggregating, posting, and/or disseminating a person’s demographic, personal, or private data without express permission.
-* No posting and/or disseminating a person’s posts, including screen captures or any other content without express permission.
-* No posting or disseminating libel, slander, or other disinformation.
+* Do not participate in xenophobia or violent nationalism.
+
+### Harassment and privacy violation
+
+* Do not contribute to behaviour intended to stalk, harass, or intimidate other users.
+* Do not continue to engage with a user that has specifically asked you to stop, whether or not they have blocked or muted you. Do not ask others to engage with them on your behalf.
+* Do not participate in aggregating, posting, and/or disseminating a person’s demographic, personal, or private data without express permission.
+* Do not post or circulate disseminate a person’s posts, including screen captures or any other content, without express permission, unless to protect others from the bad behaviour displayed.
+* Do not post or circulate libel, slander, or other known disinformation.
+
+### Harmful and/or adult content
+
+* Do not distribute sexual or violent imagery without a Content Warning.
+* Do not distribute any sexualized depictions of minors, in any way. (This includes drawings and 3D renders.)
+
+## Where does this CoC apply and who does this CoC apply to?
 
-## Where does this CoC apply and who does this CoC apply to? 
 This code of conduct applies to all official public discussion forums related to this project, including but not limited to public chat, forums, and comments on code.
 
 This code applies to any member of the community participating in discussions, contributing to the project, or otherwise discussing the project publicly.
 
 It also would apply in the future main Florence instance. Members of the instance must agree to the CoC.
```

Comment:
Will there just be one official server? If not might be better to say "would apply for any future supported Florence instances"

Reply:
Resolving on the grounds you have collectively seen this comment